README.md

```diff
@@ -8,3 +8,8 @@ git submodule update --init --recursive
 ```
 
 such that you have them matching this PCL version.
+
+## Install dependency
+```
+sudo apt install libbullet-dev
```

Comment:
why do we need this?

Reply:
yup actually needed by tf2_bullet here: https://github.com/ethz-asl/geometry2/blob/735caa6a8ad498f6291d5f742ba0fe2c1a900f33/tf2_bullet/package.xml#L17  :)